src/components/channel/ChannelAutocomplete.vue

```diff
@@ -47,7 +47,7 @@ export default {
                 })
                 .then(({ data }) => {
                     this.searchResults = data.map((d) => ({
-                        text: this.getChannelName(d),
+                        text: (d.name.includes(`${d.english_name || d.name}`) ? this.getChannelName(d) : `${this.getChannelName(d)} / ${this.getInverseName(d)}`) + ` (${d.id})`,
```

Comment:
hmm, if we wanted both, it would be rather simpler to just do 

```js
`${d.english_name || ""} ${d.name} (${d.id})`

```

then delete both getChannelName and getInverseName

Reply:
Fair enough, I was trying to only show name if it contained English name but the method was pretty flawed anyway (and somewhat contradicts my point of fixing name searches)